### PR TITLE
Fix #31: Make the hint text bolder.

### LINKED ISSUE #31: [sortedm2m] Make widget hint bolder
The help text *"Choose items and order by drag & drop"* in the sortedm2m widget is helpful but hard to see. 

Users have suggested to make it more evident.

### PR DESCRIPTION
**Pull request by rohithasrk**
This PR makes the hint-text "Choose items and order by drag & drop." bolder by changing add a little css to the admin.css file.

### CODE REVIEW

**Comment by rohithasrk** [comment]
@nemesisdesign : Please review.

File: django_netjsonconfig/static/django-netjsonconfig/css/admin.css [hunk 1]
```diff
@@ -96,3 +96,7 @@
 #container .help a,
 #netjsonconfig-hint a{ text-decoration: underline }
 #netjsonconfig-hint{ width: auto }
+
+p.help{
```
Comment (nemesisdesign) [inline django_netjsonconfig/static/django-netjsonconfig/css/admin.css]: We just wanted the text "Choose items and order by drag & drop" to be bold, not every help text.

Also, making it bolder would probably not be enough to make it more evident, we should make it darker.

Comment (nemesisdesign) [inline django_netjsonconfig/static/django-netjsonconfig/css/admin.css]: Yes, use `#333` as it's also used elsewhere in the default django-admin style.

Comment (rohithasrk) [inline django_netjsonconfig/static/django-netjsonconfig/css/admin.css]: Squashed and re pushed the commits. Please check

**Pull request merged**